### DISCUSSION TITLE
Unable to make 'notes' not be a draft?

### DISCUSSION THREAD

**Discussion by Kurru** [community/community#20180] [Projects and Issues]
Unable to make 'notes' not be a draft.

I "add item" to a new style project, but they show up as drafts? This made me think they were unsaved. Am I meant to promote all of these to issues? 

We don't use Issues for many projects, we enjoyed the previous projects support for 'notes' instead of issues.

**Comment by 2percentsilk** [comment]
Hi @Kurru,

Can you tell me a little bit more about why you think drafts are unsaved? They function very similarly to notes in the projects (classic) and are meant to be a lightweight way for you to write down thoughts that you aren't ready to promote to a full-fledged issue yet.

**Comment by Kurru** [comment]
I guess it was a first time use thing? I didn't think of 'drafts' meaning something "real" but rather "soon will be created", so I was worried to refresh or click away. 

Separately, since we don't tend to use issues for many of our projects, them being called 'draft' feels like they're "not committed" yet. The word "draft" doesn't serve any purpose with how we use products, and instead requires us to tell people "ignore 'draft', that doesn't mean anything for us". 

If draft was instead removed from the ui, or called 'note' again, this would remove the concern/distraction.

**Comment by morgzz** [comment]
I agree. I used ublock origin, to remove the verbiage. It likely will have negative consequences, but I'm cool with that.
